StarryExpanse/EverPresent/StrangerController.h

```diff
@@ -22,7 +22,8 @@ class STARRYEXPANSE_API AStrangerController : public APlayerController {
   void Interact();
   
   UFUNCTION()
-  FHitResult CastInteractionRay(bool &gotHit, bool &hadError, float xCenterOffset, float yCenterOffset);
+  FHitResult CastInteractionRay(bool &gotHit, bool &hadError, FVector worldLocation,
```

Comment:
Can we delete the unused hadError, or is it required for Blueprint use?

Reply:
Yes we can. I was using it to indicate an error that you moved outside of this function.